client/my-sites/plan-features/actions.jsx

```diff
@@ -58,7 +58,7 @@ const PlanFeaturesActions = ( {
 	if ( current && ! isInSignup ) {
 		upgradeButton = (
 			<Button className={ classes } href={ manageHref } disabled={ ! manageHref }>
-				{ canPurchase ? translate( 'Manage Plan' ) : translate( 'View Plan' ) }
+				{ canPurchase ? translate( 'Manage plan' ) : translate( 'View plan' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.